src/components/NavBar.tsx

```diff
@@ -1,15 +1,59 @@
 import React from 'react';
-import { AppBar, Tabs, Tab } from '@material-ui/core';
-import { Link } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router-dom';
+import { Layout, Menu } from 'antd';
+const { Header, Footer, Sider, Content } = Layout;
 
 const NavBar: React.FC = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const path: string = location.pathname;
+
+  function pathnameToKey(pathname: string) {
```

Comment:
💭 This could also be a StringMap (I think that's a standard type?)

Reply:
oo Good idea!